RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.1
+### Update
+  - Added 2 catalog parameters in AssemblyUtilImpl
+  - Updated `save_assemblies_from_fastas` to parallelize `import_fasta_mass` function
+  - Added unit/integration tests for parallelization
```

Comment:
Usually no need to discuss tests in release notes

Reply:
Updated!

Reply:
:+1: